python/pip_install/pip_repository.bzl

```diff
@@ -125,20 +126,65 @@ def _get_toolchain_unix_cflags(rctx):
     if not is_standalone_interpreter(rctx, rctx.attr.python_interpreter_target):
         return []
 
+    include_paths = []
+
+    is_clang = False
+    if c_compiler:
```

Comment:
could you add some commenting about the intent/approach here? it seems like it's "call the compiler to find out what includes it expects by default" or something